<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en">

<!-- Mirrored from arxiv.org/list/q-bio/new by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 29 Mar 2019 22:40:35 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
<title>Quantitative Biology  authors/titles &quot;new&quot;</title>
<link rel="shortcut icon" href="../../favicon.ico" type="image/x-icon" />
<link rel="stylesheet" type="text/css" media="screen" href="../../css/arXivfecf.css?v=20190306" />
<link rel="stylesheet" type="text/css" media="screen" href="../../bibex/bibex2785.css?20181009">
<link rel="stylesheet" href="../../../maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"> 
<link rel="alternate" type="application/rss+xml" title="Quantitative Biology " href="http://arxiv.org/rss/q-bio"/>
<script src="https://arxiv.org/js/mathjaxToggle.min.js" type="text/javascript"></script>

<!-- Piwik -->
<script type="text/javascript">
var _paq = _paq || [];
_paq.push(["setDomains", ["*.arxiv.org"]]);
_paq.push(['trackPageView']);
_paq.push(['enableLinkTracking']);
(function()
{ var u="../../../webanalytics.library.cornell.edu/index.html"; _paq.push(['setTrackerUrl', u+'piwik.php']); _paq.push(['setSiteId', 538]); var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0]; g.type='text/javascript'; g.async=true; g.defer=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s); }
)();
</script>
<!-- End Piwik Code -->


</head>
<body class="with-cu-identity">

<noscript><img src="../../../webanalytics.library.cornell.edu/piwikf25d.gif?idsite=538&amp;rec=1" style="border:0;" alt="" /></noscript>

<div id="cu-identity">
<div id="cu-logo">
<a href="../../../www.cornell.edu/index.html"><img src="../../../static.arxiv.org/icons/cu/cornell-reduced-white-SMALL.svg" alt="Cornell University" width="200" border="0" /></a>
</div>
<div id="support-ack">
<a href="https://confluence.cornell.edu/x/ALlRF">We gratefully acknowledge support from<br /> the Simons Foundation and member institutions.</a>
</div>
</div>
<div id="header">
<h1><a href="../../index.html">arXiv.org</a> &gt; <a href="recent.html">q-bio</a></h1>
<div id="search">
<form id="search-arxiv" method="GET" action="https://arxiv.org/search">
                
<div class="wrapper-search-arxiv">
<input class="keyword-field" type="text" name="query" placeholder="Search or Article ID"/>

<div class="filter-field">
<select name="searchtype">
<option value="all">All fields</option>
<option value="title">Title</option>
<option value="author">Author(s)</option>
<option value="abstract">Abstract</option>
<option value="comments">Comments</option>
<option value="journal_ref">Journal reference</option>
<option value="acm_class">ACM classification</option>
<option value="msc_class">MSC classification</option>
<option value="report_num">Report number</option>
<option value="paper_id">arXiv identifier</option>
<option value="doi">DOI</option>
<option value="orcid">ORCID</option>
<option value="author_id">arXiv author ID</option>
<option value="help">Help pages</option>
<option value="full_text">Full text</option></select>
</div>
<input class="btn-search-arxiv" value="" type="submit">
<div class="links">(<a href="../../help.html">Help</a> | <a href="https://arxiv.org/search/advanced">Advanced search</a>)</div>
</div>
<input type="hidden" name="source" value="header">
</form>
</div>
</div>
<div id="content">
<div id="dlpage">
<h1>Quantitative Biology </h1>
<h2>New submissions</h2>
<div class="list-dateline">Submissions received from  Wed 27 Mar 19  to  Thu 28 Mar 19, announced Fri, 29 Mar 19</div>
<ul>
<li><a href="https://arxiv.org/list/q-bio/new?skip=0&amp;show=2000">New submissions</a></li>
<li><a href="#item11">Cross-lists</a></li>
<li><a href="#item12">Replacements</a></li>
</ul>
<small>[ total of 20 entries:  <b>1-20</b>  ]</small><br />
<small>[ showing up to 2000 entries per page:  <a href="https://arxiv.org/list/q-bio/new?skip=0&amp;show=1000">fewer</a> |  <font color="#999999">more</font> ]</small><br />
<h3>New submissions for Fri, 29 Mar 19</h3>
<dl>
<dt><a name="item1">[1]</a>&nbsp;  <span class="list-identifier"><a href="https://arxiv.org/abs/1903.11597" title="Abstract">arXiv:1903.11597</a> [<a href="https://arxiv.org/pdf/1903.11597" title="Download PDF">pdf</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title mathjax">
<span class="descriptor">Title:</span> PCL enzymatic hydrolysis: a mechanistic study
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Almeida%2C+B+C">Beatriz C. Almeida</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Figueiredo%2C+P">Pedro Figueiredo</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Carvalho%2C+A+T+P">Alexandra T. P. Carvalho</a> (CNC, Center for Neuroscience and Cell Biology, Institute for Interdisciplinary Research (IIIUC), University of Coimbra, Coimbra (Portugal))
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Biomolecules (q-bio.BM)</span>

</div>
<p class="mathjax">Accumulation of plastic waste is a major environmental problem. Enzymes,
particularly esterases, play an important role in the biodegradation of
polyesters. These enzymes are usually only active on aliphatic polyesters, but
a few have showed catalytic activity for semi-aromatic polyesters. Due to the
importance of these processes, an atomic level characterization of how common
polyesters are degraded by esterases is necessary. Hereby, we present a
Molecular dynamics (MD) and Quantum Mechanics/Molecular Mechanics (QM/MM) MD
study of the hydrolysis of a model of polycaprolactone (PCL), one of the most
widely used biomaterials, by the thermophilic esterase from the archaeon
Archaeoglobus fulgidus (AfEST). This enzyme is particularly interesting because
it can withstand temperatures well above the glass transition of many
polyesters. Our insights about the reaction mechanism are important for the
design of customized enzymes able to degrade different synthetic polyesters.
</p>
</div>
</dd>
<dt><a name="item2">[2]</a>&nbsp;  <span class="list-identifier"><a href="https://arxiv.org/abs/1903.11627" title="Abstract">arXiv:1903.11627</a> [<a href="https://arxiv.org/pdf/1903.11627" title="Download PDF">pdf</a>, <a href="https://arxiv.org/ps/1903.11627" title="Download PostScript">ps</a>, <a href="https://arxiv.org/format/1903.11627" title="Other formats">other</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title mathjax">
<span class="descriptor">Title:</span> Genetic drift in range expansions is very sensitive to density feedback  in dispersal and growth
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Birzu%2C+G">Gabriel Birzu</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Matin%2C+S">Sakib Matin</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Hallatschek%2C+O">Oskar Hallatschek</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Korolev%2C+K+S">Kirill S. Korolev</a>
</div>
<div class="list-comments mathjax">
<span class="descriptor">Comments:</span> 36 pages, 5 figures, and 1 table
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Populations and Evolution (q-bio.PE)</span>; Statistical Mechanics (cond-mat.stat-mech); Biological Physics (physics.bio-ph)

</div>
<p class="mathjax">Theory predicts rapid genetic drift during invasions, yet many expanding
populations maintain high genetic diversity. We find that genetic drift is
dramatically suppressed when dispersal rates increase with the population
density because many more migrants from the diverse, high-density regions
arrive at the expansion edge. When density-dependence is weak or negative, the
effective population size of the front scales only logarithmically with the
carrying capacity. The dependence, however, switches to a sublinear power law
and then to a linear increase as the density-dependence becomes strongly
positive. We develop a unified framework revealing that the transitions between
different regimes of diversity loss are controlled by a single, universal
parameter: the ratio of the expansion velocity to the geometric mean of
dispersal and growth rates at expansion edge. Our results suggest that positive
density-dependence could dramatically alter evolution in expanding populations
even when its contributions to the expansion velocity is small.
</p>
</div>
</dd>
<dt><a name="item3">[3]</a>&nbsp;  <span class="list-identifier"><a href="https://arxiv.org/abs/1903.11670" title="Abstract">arXiv:1903.11670</a> [<a href="https://arxiv.org/pdf/1903.11670" title="Download PDF">pdf</a>, <a href="https://arxiv.org/ps/1903.11670" title="Download PostScript">ps</a>, <a href="https://arxiv.org/format/1903.11670" title="Other formats">other</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title mathjax">
<span class="descriptor">Title:</span> The minimum value of the Colless index
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Coronado%2C+T+M">Tom&#xe1;s M. Coronado</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Rossell%C3%B3%2C+F">Francesc Rossell&#xf3;</a>
</div>
<div class="list-comments mathjax">
<span class="descriptor">Comments:</span> 13 pages
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Populations and Evolution (q-bio.PE)</span>; Discrete Mathematics (cs.DM); Combinatorics (math.CO)

</div>
<p class="mathjax">The Colless index is one of the oldest and most widely used balance indices
for rooted bifurcating trees. Despite its popularity, its minimum value on the
space $\mathcal{T}_n$ of rooted bifurcating trees with $n$ leaves is only known
when $n$ is a power of 2. In this paper we fill this gap in the literature, by
providing a formula that computes, for each $n$, the minimum Colless index on
$\mathcal{T}_n$, and characterizing those trees where this minimum value is
reached.
</p>
</div>
</dd>
<dt><a name="item4">[4]</a>&nbsp;  <span class="list-identifier"><a href="https://arxiv.org/abs/1903.11797" title="Abstract">arXiv:1903.11797</a> [<a href="https://arxiv.org/pdf/1903.11797" title="Download PDF">pdf</a>, <a href="https://arxiv.org/format/1903.11797" title="Other formats">other</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title mathjax">
<span class="descriptor">Title:</span> Estimating effective population size changes from preferentially sampled  genetic sequences
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Karcher%2C+M+D">Michael D. Karcher</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Suchard%2C+M+A">Marc A. Suchard</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Dudas%2C+G">Gytis Dudas</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Minin%2C+V+N">Vladimir N. Minin</a>
</div>
<div class="list-comments mathjax">
<span class="descriptor">Comments:</span> 47 pages
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Populations and Evolution (q-bio.PE)</span>; Methodology (stat.ME)

</div>
<p class="mathjax">Coalescent theory combined with statistical modeling allows us to estimate
effective population size fluctuations from molecular sequences of individuals
sampled from a population of interest. When sequences are sampled serially
through time and the distribution of the sampling times depends on the
effective population size, explicit statistical modeling of sampling times
improves population size estimation. Previous work assumed that the genealogy
relating sampled sequences is known and modeled sampling times as an
inhomogeneous Poisson process with log-intensity equal to a linear function of
the log-transformed effective population size. We improve this approach in two
ways. First, we extend the method to allow for joint Bayesian estimation of the
genealogy, effective population size trajectory, and other model parameters.
Next, we improve the sampling time model by incorporating additional sources of
information in the form of time-varying covariates. We validate our new
modeling framework using a simulation study and apply our new methodology to
analyses of population dynamics of seasonal influenza and to the recent Ebola
virus outbreak in West Africa.
</p>
</div>
</dd>
<dt><a name="item5">[5]</a>&nbsp;  <span class="list-identifier"><a href="https://arxiv.org/abs/1903.11913" title="Abstract">arXiv:1903.11913</a> [<a href="https://arxiv.org/pdf/1903.11913" title="Download PDF">pdf</a>, <a href="https://arxiv.org/ps/1903.11913" title="Download PostScript">ps</a>, <a href="https://arxiv.org/format/1903.11913" title="Other formats">other</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title mathjax">
<span class="descriptor">Title:</span> Accessibility percolation in random fitness landscapes
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Krug%2C+J">Joachim Krug</a>
</div>
<div class="list-comments mathjax">
<span class="descriptor">Comments:</span> 19 pages, 3 figures; to appear in "Probabilistic Structures in Evolution", ed. by E. Baake and A. Wakolbinger
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Populations and Evolution (q-bio.PE)</span>; Probability (math.PR)

</div>
<p class="mathjax">The fitness landscape encodes the mapping of genotypes to fitness and
provides a succinct representation of possible trajectories followed by an
evolving population. Evolutionary accessibility is quantified by the existence
of fitness-monotonic paths connecting far away genotypes. Studies of
accessibility percolation use probabilistic fitness landscape models to explore
the emergence of such paths as a function of the initial fitness, the
parameters of the landscape or the structure of the genotype graph. This
chapter reviews these studies and discusses their implications for the
predictability of evolutionary processes.
</p>
</div>
</dd>
<dt><a name="item6">[6]</a>&nbsp;  <span class="list-identifier"><a href="https://arxiv.org/abs/1903.11915" title="Abstract">arXiv:1903.11915</a> [<a href="https://arxiv.org/pdf/1903.11915" title="Download PDF">pdf</a>, <a href="https://arxiv.org/format/1903.11915" title="Other formats">other</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title mathjax">
<span class="descriptor">Title:</span> Early deafness leads to re-shaping of global functional connectivity  beyond the auditory cortex
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Bonna%2C+K">Kamil Bonna</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Finc%2C+K">Karolina Finc</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Zimmermann%2C+M">Maria Zimmermann</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Bola%2C+%C5%81">&#x141;ukasz Bola</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Mostowski%2C+P">Piotr Mostowski</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Szul%2C+M">Maciej Szul</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Rutkowski%2C+P">Pawe&#x142; Rutkowski</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Duch%2C+W">W&#x142;odzis&#x142;aw Duch</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Marchewka%2C+A">Artur Marchewka</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Jednor%C3%B3g%2C+K">Katarzyna Jednor&#xf3;g</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Szwed%2C+M">Marcin Szwed</a>
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Neurons and Cognition (q-bio.NC)</span>

</div>
<p class="mathjax">Early sensory deprivation such as blindness or deafness shapes brain
development in multiple ways. While it is established that deprived brain areas
start to be engaged in the processing of stimuli from the remaining modalities
and in high-level cognitive tasks, some reports have also suggested the
possibility of structural and functional changes in non-deprived brain areas.
We compared resting-state functional network organization of the brain in
early-deaf adults and hearing controls by examining global network segregation
and integration. Relative to hearing controls, deaf adults exhibited an altered
modular organization, with regions of the salience network coupled with the
fronto-parietal network. They showed weaker connections between auditory and
somatomotor regions, stronger coupling between the fronto-parietal network and
several other large-scale networks (visual, memory, cingulo-opercular and
somatomotor), and an enlargement of the default mode network. Their overall
functional segregation of brain networks was also lower. Our findings suggest
that brain plasticity in deaf adults is not limited to changes in auditory
cortex but additionally alters the coupling between other large-scale networks.
These widespread functional connectivity changes may provide a mechanism for
the superior behavioral performance of the deaf in visual and attentional
tasks.
</p>
</div>
</dd>
<dt><a name="item7">[7]</a>&nbsp;  <span class="list-identifier"><a href="https://arxiv.org/abs/1903.11972" title="Abstract">arXiv:1903.11972</a> [<a href="https://arxiv.org/pdf/1903.11972" title="Download PDF">pdf</a>, <a href="https://arxiv.org/format/1903.11972" title="Other formats">other</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title mathjax">
<span class="descriptor">Title:</span> Developmental trajectory of the Caenorhabditis elegans nervous system  governs its structural organization
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Pathak%2C+A">Anand Pathak</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Chatterjee%2C+N">Nivedita Chatterjee</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Sinha%2C+S">Sitabhra Sinha</a>
</div>
<div class="list-comments mathjax">
<span class="descriptor">Comments:</span> 24 pages, 8 figures + 11 pages supplementary material
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Neurons and Cognition (q-bio.NC)</span>

</div>
<p class="mathjax">A central problem of neuroscience involves uncovering the principles
governing the organization of nervous systems which ensure robustness in brain
development. The nematode Caenorhabditis elegans, where genetically identical
individuals possess the same complement of neurons located at invariant
positions, provides us with a model organism for studying this question. In
this paper, we focus on the connection structure and spatial arrangement of the
neurons comprising the somatic neuronal network of this organism to understand
the key developmental constraints underlying the design of a simple nervous
system. We observe that lineage distance between neurons, rather than the
physical distance between their cell bodies, strongly influences the
probability of a connection between them. In addition, the length of neuronal
processes, as well as, their birth times determine connectivity. Together,
these results help connect the physical location and morphology of neurons with
the topological organization of the network. Further, neurons coordinating
activity across distinct neuronal clusters are seen to appear early which
relates the temporal hierarchy imposed by development to the functional role of
neurons. By providing a framework for a mechanistic understanding of how
development shapes a nervous system, our results complement interpretive
analysis of how efficiency and optimality govern its design. Providing crucial
insights for understanding principles that may be common across many organisms,
our study shows how the trajectory in the developmental landscape constrains
the eventual spatial and network topological organization of a nervous system.
</p>
</div>
</dd>
<dt><a name="item8">[8]</a>&nbsp;  <span class="list-identifier"><a href="https://arxiv.org/abs/1903.12009" title="Abstract">arXiv:1903.12009</a> [<a href="https://arxiv.org/pdf/1903.12009" title="Download PDF">pdf</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title mathjax">
<span class="descriptor">Title:</span> Deciphering the underlying mechanisms of the pharyngeal motions in  Caenorhabditis elegans
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Sherman%2C+D">Dana Sherman</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Harel%2C+D">David Harel</a>
</div>
<div class="list-comments mathjax">
<span class="descriptor">Comments:</span> 32 pages, 7 figures
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Neurons and Cognition (q-bio.NC)</span>; Tissues and Organs (q-bio.TO)

</div>
<p class="mathjax">The pharynx of the nematode Caenorhabditis elegans is a neuromuscular pump
that exhibits two typical motions: pumping and peristalsis. While the dynamics
of these motions are well characterized, the underlying mechanisms generating
most of them are not known. In this paper, we propose comprehensive and
detailed mechanisms that can explain the various observed dynamics of the
different pharyngeal areas: the dynamics of the pumping muscles - corpus,
anterior isthmus, and terminal bulb - and the peristalsis dynamics of the
posterior isthmus muscles. While the suggested mechanisms are consistent with
all available relevant data, the assumptions on which they are based and the
open questions they raise could point at additional interesting research
directions on the C. elegans pharynx. We are hoping that appropriate
experiments on the nematode will eventually corroborate our results, and
improve our understanding of the functioning of the C. elegans pharynx, and
possibly of the mammalian digestive system.
</p>
</div>
</dd>
<dt><a name="item9">[9]</a>&nbsp;  <span class="list-identifier"><a href="https://arxiv.org/abs/1903.12138" title="Abstract">arXiv:1903.12138</a> [<a href="https://arxiv.org/pdf/1903.12138" title="Download PDF">pdf</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title mathjax">
<span class="descriptor">Title:</span> Overt and covert paths for sound in the auditory system of mammals
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Auriol%2C+B+M">Bernard M.Auriol</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Auriol%2C+B">Benjamin Auriol</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=B%C3%A9ard%2C+J">J&#xe9;rome B&#xe9;ard</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Bib%C3%A9%2C+B">Bernard Bib&#xe9;</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Broto%2C+J">Jean-Marc Broto</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Descouens%2C+D">Didier Descouens</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Durand%2C+L+J+S">Lise J.S. Durand</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Florens%2C+J">Jean-Pierre Florens</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Garcia%2C+F">Fr&#xe9;d&#xe9;rick Garcia</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Joiner%2C+E+G">Elizabeth G. Joiner</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Libes%2C+B">Bernard Libes</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Pergent%2C+P">Philippe Pergent</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Ruiz%2C+R">Robert Ruiz</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Thalamas%2C+C">Claire Thalamas</a>
</div>
<div class="list-comments mathjax">
<span class="descriptor">Comments:</span> preprint
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Tissues and Organs (q-bio.TO)</span>

</div>
<p class="mathjax">Current scientific consensus holds that sound is transmitted, solely
mechanically, from the tympanum to the cochlea via ossicles. But this theory
does not explain the hearing extreme quality regarding high frequencies in
mammals. So, we propose a bioelectronic pathway (the covert path) that is
complementary to the overt path.. We demonstrate experimentally that the
tympanum produces piezoelectric potentials isochronous to acoustic vibrations
thanks to its collagen fibers and that their amplitude increases along with the
frequency and level of the vibrations. This finding supports the existence of
an electrical pathway, specialized in transmitting high-frequency sounds, that
works in unison with the mechanical pathway. A bio-organic triode, similar to a
field effect transistor, is the key mechanism of our hypothesized pathway. We
present evidence that any deficiency along this pathway produces hearing
impairment. By augmenting the classical theory of sound transmission, our
discovery offers new perspectives for research into both normal and
pathological audition and may contribute to an understanding of genetic and
physiological problems of hearing.
</p>
</div>
</dd>
<dt><a name="item10">[10]</a>&nbsp;  <span class="list-identifier"><a href="https://arxiv.org/abs/1903.12155" title="Abstract">arXiv:1903.12155</a> [<a href="https://arxiv.org/pdf/1903.12155" title="Download PDF">pdf</a>, <a href="https://arxiv.org/format/1903.12155" title="Other formats">other</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title mathjax">
<span class="descriptor">Title:</span> Theta-gamma cross-frequency coupling enables covariance between distant  brain regions
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Akao%2C+A">Akihiko Akao</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Shirasaka%2C+S">Sho Shirasaka</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Jimbo%2C+Y">Yasuhiko Jimbo</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Ermentrout%2C+B">Bard Ermentrout</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Kotani%2C+K">Kiyoshi Kotani</a>
</div>
<div class="list-comments mathjax">
<span class="descriptor">Comments:</span> 13 pages, 4 figures
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Neurons and Cognition (q-bio.NC)</span>; Pattern Formation and Solitons (nlin.PS)

</div>
<p class="mathjax">Cross-frequency coupling (CFC) is thought to play an important role in
communication across distant brain regions. However, neither the mechanism of
its generation nor the influence on the underlying spiking dynamics is well
understood. Here, we investigate the dynamics of two interacting distant
neuronal modules coupled by inter-regional long-range connections. Each
neuronal module comprises an excitatory and inhibitory population of quadratic
integrate-and-fire neurons connected locally with conductance-based synapses.
The two modules are coupled reciprocally with delays that represent the
long-range conduction time. We applied the Ott-Antonsen ansatz to reduce the
spiking dynamics to the corresponding mean field equations as a small set of
delay differential equations. Bifurcation analysis on these mean field
equations shows inter-regional conduction delay is sufficient to produce CFC
via a torus bifurcation. Spike correlation analysis during the CFC revealed
that several local clusters exhibit synchronized firing in gamma-band
frequencies. These clusters exhibit locally decorrelated firings between the
cluster pairs within the same population. In contrast, the clusters exhibit
long-range gamma-band cross-covariance between the distant clusters that have
similar firing frequency. The interactions of the different gamma frequencies
produce a beat leading to population-level CFC. We analyzed spike counts in
relation to the phases of the macroscopic fast and slow oscillations and found
population spike counts vary with respect to macroscopic phases. Such firing
phase preference accompanies a phase window with high spike count and low Fano
factor, which is suitable for a population rate code. Our work suggests the
inter-regional conduction delay plays a significant role in the emergence of
CFC and the underlying spiking dynamics may support long-range communication
and neural coding.
</p>
</div>
</dd>
</dl>
<h3>Cross-lists for Fri, 29 Mar 19</h3>
<dl>
<dt><a name="item11">[11]</a>&nbsp;  <span class="list-identifier"><a href="https://arxiv.org/abs/1903.11696" title="Abstract">arXiv:1903.11696</a> (cross-list from stat.ML) [<a href="https://arxiv.org/pdf/1903.11696" title="Download PDF">pdf</a>, <a href="https://arxiv.org/format/1903.11696" title="Other formats">other</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title mathjax">
<span class="descriptor">Title:</span> Stable prediction with radiomics data
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="https://arxiv.org/search/stat?searchtype=author&amp;query=Peeters%2C+C+F+W">Carel F.W. Peeters</a>, 
<a href="https://arxiv.org/search/stat?searchtype=author&amp;query=%C3%9Cbelh%C3%B6r%2C+C">Caroline &#xdc;belh&#xf6;r</a>, 
<a href="https://arxiv.org/search/stat?searchtype=author&amp;query=Mes%2C+S+W">Steven W. Mes</a>, 
<a href="https://arxiv.org/search/stat?searchtype=author&amp;query=Martens%2C+R">Roland Martens</a>, 
<a href="https://arxiv.org/search/stat?searchtype=author&amp;query=Koopman%2C+T">Thomas Koopman</a>, 
<a href="https://arxiv.org/search/stat?searchtype=author&amp;query=de+Graaf%2C+P">Pim de Graaf</a>, 
<a href="https://arxiv.org/search/stat?searchtype=author&amp;query=van+Velden%2C+F+H+P">Floris H.P. van Velden</a>, 
<a href="https://arxiv.org/search/stat?searchtype=author&amp;query=Boellaard%2C+R">Ronald Boellaard</a>, 
<a href="https://arxiv.org/search/stat?searchtype=author&amp;query=Castelijns%2C+J+A">Jonas A. Castelijns</a>, 
<a href="https://arxiv.org/search/stat?searchtype=author&amp;query=Beest%2C+D+E+t">Dennis E. te Beest</a>, 
<a href="https://arxiv.org/search/stat?searchtype=author&amp;query=Heymans%2C+M+W">Martijn W. Heymans</a>, 
<a href="https://arxiv.org/search/stat?searchtype=author&amp;query=van+de+Wiel%2C+M+A">Mark A. van de Wiel</a>
</div>
<div class="list-comments mathjax">
<span class="descriptor">Comments:</span> 52 pages: 14 pages Main Text and 38 pages of Supplementary Material
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Machine Learning (stat.ML)</span>; Machine Learning (cs.LG); Image and Video Processing (eess.IV); Quantitative Methods (q-bio.QM); Applications (stat.AP); Methodology (stat.ME)

</div>
<p class="mathjax">Motivation: Radiomics refers to the high-throughput mining of quantitative
features from radiographic images. It is a promising field in that it may
provide a non-invasive solution for screening and classification. Standard
machine learning classification and feature selection techniques, however, tend
to display inferior performance in terms of (the stability of) predictive
performance. This is due to the heavy multicollinearity present in radiomic
data. We set out to provide an easy-to-use approach that deals with this
problem.
<br />Results: We developed a four-step approach that projects the original
high-dimensional feature space onto a lower-dimensional latent-feature space,
while retaining most of the covariation in the data. It consists of (i)
penalized maximum likelihood estimation of a redundancy filtered correlation
matrix. The resulting matrix (ii) is the input for a maximum likelihood factor
analysis procedure. This two-stage maximum-likelihood approach can be used to
(iii) produce a compact set of stable features that (iv) can be directly used
in any (regression-based) classifier or predictor. It outperforms other
classification (and feature selection) techniques in both external and internal
validation settings regarding survival in squamous cell cancers.
</p>
</div>
</dd>
</dl>
<h3>Replacements for Fri, 29 Mar 19</h3>
<dl>
<dt><a name="item12">[12]</a>&nbsp;  <span class="list-identifier"><a href="https://arxiv.org/abs/1607.04656" title="Abstract">arXiv:1607.04656</a> (replaced) [<a href="https://arxiv.org/pdf/1607.04656" title="Download PDF">pdf</a>, <a href="https://arxiv.org/format/1607.04656" title="Other formats">other</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title mathjax">
<span class="descriptor">Title:</span> Stochastic dynamics of three competing clones: Conditions and times for  invasion, coexistence and fixation
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Billiard%2C+S">Sylvain Billiard</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Smadi%2C+C">Charline Smadi</a>
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Populations and Evolution (q-bio.PE)</span>

</div>
</div>
</dd>
<dt><a name="item13">[13]</a>&nbsp;  <span class="list-identifier"><a href="https://arxiv.org/abs/1703.01357" title="Abstract">arXiv:1703.01357</a> (replaced) [<a href="https://arxiv.org/pdf/1703.01357" title="Download PDF">pdf</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title mathjax">
<span class="descriptor">Title:</span> A Computational Model of Systems Memory Reconsolidation
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Helfer%2C+P">Peter Helfer</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Shultz%2C+T+R">Thomas R. Shultz</a>
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Neurons and Cognition (q-bio.NC)</span>

</div>
</div>
</dd>
<dt><a name="item14">[14]</a>&nbsp;  <span class="list-identifier"><a href="https://arxiv.org/abs/1808.03091" title="Abstract">arXiv:1808.03091</a> (replaced) [<a href="https://arxiv.org/pdf/1808.03091" title="Download PDF">pdf</a>, <a href="https://arxiv.org/format/1808.03091" title="Other formats">other</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title mathjax">
<span class="descriptor">Title:</span> Breakdown of effective temperature, power law interactions and  self-propulsion in a momentum conserving active fluid
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="https://arxiv.org/search/cond-mat?searchtype=author&amp;query=Vishen%2C+A+S">Amit Singh Vishen</a>, 
<a href="https://arxiv.org/search/cond-mat?searchtype=author&amp;query=Prost%2C+J">Jacques Prost</a>, 
<a href="https://arxiv.org/search/cond-mat?searchtype=author&amp;query=Rao%2C+M">Madan Rao</a>
</div>
<div class="list-comments mathjax">
<span class="descriptor">Comments:</span> Modified the fluctuating force analysis and removed the enzyme dynamics calculation
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Soft Condensed Matter (cond-mat.soft)</span>; Biological Physics (physics.bio-ph); Quantitative Methods (q-bio.QM)

</div>
</div>
</dd>
<dt><a name="item15">[15]</a>&nbsp;  <span class="list-identifier"><a href="https://arxiv.org/abs/1808.07785" title="Abstract">arXiv:1808.07785</a> (replaced) [<a href="https://arxiv.org/pdf/1808.07785" title="Download PDF">pdf</a>, <a href="https://arxiv.org/ps/1808.07785" title="Download PostScript">ps</a>, <a href="https://arxiv.org/format/1808.07785" title="Other formats">other</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title mathjax">
<span class="descriptor">Title:</span> Inter-state switching in stochastic gene expression: Exact solution, an  adiabatic limit and oscillations in molecular distributions
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Petrosyan%2C+K+G">K.G. Petrosyan</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Zacharovas%2C+V">V. Zacharovas</a>
</div>
<div class="list-comments mathjax">
<span class="descriptor">Comments:</span> 6 pages, 6 figures
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Molecular Networks (q-bio.MN)</span>; Statistical Mechanics (cond-mat.stat-mech)

</div>
</div>
</dd>
<dt><a name="item16">[16]</a>&nbsp;  <span class="list-identifier"><a href="https://arxiv.org/abs/1810.11195" title="Abstract">arXiv:1810.11195</a> (replaced) [<a href="https://arxiv.org/pdf/1810.11195" title="Download PDF">pdf</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title mathjax">
<span class="descriptor">Title:</span> Prey selection of Amur tigers in relation to the spatiotemporal overlap  with prey across the Sino-Russian border
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Dou%2C+H">Hailong Dou</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Yang%2C+H">Haitao Yang</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Smith%2C+J+L+D">James L.D. Smith</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Feng%2C+L">Limin Feng</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Wang%2C+T">Tianming Wang</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Ge%2C+J">Jianping Ge</a>
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Populations and Evolution (q-bio.PE)</span>

</div>
</div>
</dd>
<dt><a name="item17">[17]</a>&nbsp;  <span class="list-identifier"><a href="https://arxiv.org/abs/1812.06400" title="Abstract">arXiv:1812.06400</a> (replaced) [<a href="https://arxiv.org/pdf/1812.06400" title="Download PDF">pdf</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title mathjax">
<span class="descriptor">Title:</span> Automated discovery of GPCR bioactive ligands
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Raschka%2C+S">Sebastian Raschka</a>
</div>
<div class="list-journal-ref">
<span class="descriptor">Journal-ref:</span> Current Opinion in Structural Biology 2019, 55:17-24
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Biomolecules (q-bio.BM)</span>

</div>
</div>
</dd>
<dt><a name="item18">[18]</a>&nbsp;  <span class="list-identifier"><a href="https://arxiv.org/abs/1901.07298" title="Abstract">arXiv:1901.07298</a> (replaced) [<a href="https://arxiv.org/pdf/1901.07298" title="Download PDF">pdf</a>, <a href="https://arxiv.org/ps/1901.07298" title="Download PostScript">ps</a>, <a href="https://arxiv.org/format/1901.07298" title="Other formats">other</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title mathjax">
<span class="descriptor">Title:</span> Online Estimation of Multiple Dynamic Graphs in Pattern Sequences
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="https://arxiv.org/search/stat?searchtype=author&amp;query=Gaudreault%2C+J">Jimmy Gaudreault</a>, 
<a href="https://arxiv.org/search/stat?searchtype=author&amp;query=Saxena%2C+A">Arunabh Saxena</a>, 
<a href="https://arxiv.org/search/stat?searchtype=author&amp;query=Shimazaki%2C+H">Hideaki Shimazaki</a>
</div>
<div class="list-comments mathjax">
<span class="descriptor">Comments:</span> 8 pages, 4 figures v2: IJCNN 2019, results unchanged
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Machine Learning (stat.ML)</span>; Disordered Systems and Neural Networks (cond-mat.dis-nn); Machine Learning (cs.LG); Neural and Evolutionary Computing (cs.NE); Neurons and Cognition (q-bio.NC)

</div>
</div>
</dd>
<dt><a name="item19">[19]</a>&nbsp;  <span class="list-identifier"><a href="https://arxiv.org/abs/1902.07787" title="Abstract">arXiv:1902.07787</a> (replaced) [<a href="https://arxiv.org/pdf/1902.07787" title="Download PDF">pdf</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title mathjax">
<span class="descriptor">Title:</span> Biophysics at the coffee shop: lessons learned working with George Oster
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Igoshin%2C+O">Oleg Igoshin</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Chen%2C+J">Jing Chen</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Xing%2C+J">Jianhua Xing</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Liu%2C+J">Jian Liu</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Elston%2C+T+C">Timothy C. Elston</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Grabe%2C+M">Michael Grabe</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Kim%2C+K+S">Kenneth S. Kim</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Nirody%2C+J">Jasmine Nirody</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Rangamani%2C+P">Padmini Rangamani</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Sun%2C+S">Sean Sun</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Wang%2C+H">Hongyun Wang</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Wolgemuth%2C+C">Charles Wolgemuth</a>
</div>
<div class="list-comments mathjax">
<span class="descriptor">Comments:</span> 22 pages, 3 figures, accepted in Molecular Biology of the Cell
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Other Quantitative Biology (q-bio.OT)</span>

</div>
</div>
</dd>
<dt><a name="item20">[20]</a>&nbsp;  <span class="list-identifier"><a href="https://arxiv.org/abs/1903.05979" title="Abstract">arXiv:1903.05979</a> (replaced) [<a href="https://arxiv.org/pdf/1903.05979" title="Download PDF">pdf</a>, <a href="https://arxiv.org/format/1903.05979" title="Other formats">other</a>]</span></dt>
<dd>
<div class="meta">
<div class="list-title mathjax">
<span class="descriptor">Title:</span> The Frequent Complete Subgraphs in the Human Connectome
</div>
<div class="list-authors">
<span class="descriptor">Authors:</span> 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Fellner%2C+M">Mate Fellner</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Varga%2C+B">Balint Varga</a>, 
<a href="https://arxiv.org/search/q-bio?searchtype=author&amp;query=Grolmusz%2C+V">Vince Grolmusz</a>
</div>
<div class="list-subjects">
<span class="descriptor">Subjects:</span> <span class="primary-subject">Neurons and Cognition (q-bio.NC)</span>

</div>
</div>
</dd>
</dl>
<ul>
<li><a href="https://arxiv.org/list/q-bio/new?skip=0&amp;show=2000">New submissions</a></li>
<li><a href="#item11">Cross-lists</a></li>
<li><a href="#item12">Replacements</a></li>
</ul>
<small>[ total of 20 entries:  <b>1-20</b>  ]</small><br />
<small>[ showing up to 2000 entries per page:  <a href="https://arxiv.org/list/q-bio/new?skip=0&amp;show=1000">fewer</a> |  <font color="#999999">more</font> ]</small><br />
</div>
<br/><small><a id="mathjax_toggle" href="javascript:setMathjaxCookie()">Disable MathJax</a> (<a href="https://arxiv.org/help/mathjax/">What is MathJax?</a>)</small><script type="text/javascript" language="javascript">mathjaxToggle();</script>

<hr />
<p>Links to:
<a href="../../index.html" accesskey="a">arXiv</a>, 
<a href="https://arxiv.org/form/q-bio">form interface</a>,
<a href="https://arxiv.org/find/q-bio">find</a>,
<a href="../../archive/q-bio.html">q-bio</a>, <a href="recent.html">recent</a>, <a href="https://arxiv.org/list/q-bio/1903">1903</a>,
<a href="../../help/contact.html">contact</a>,
<a href="https://arxiv.org/help/" accesskey="h"><span class="accesskey">h</span>elp</a>&nbsp;
<small>(<a href="https://arxiv.org/help/accesskeys">Access key</a> information)</small>
</p>
<hr />
</div>
</div>
  <footer style="clear: both;">
    <div class="columns is-desktop" role="navigation" aria-label="Secondary" style="margin: -0.75em -0.75em 0.75em -0.75em">
      <!-- Macro-Column 1 -->
      <div class="column" style="padding: 0;">
        <div class="columns">
          <div class="column">
            <ul style="list-style: none; line-height: 2;">
              <li><a href="../../about.html">About arXiv</a></li>
              <li><a href="../../about/people/leadership_team.html">Leadership Team</a></li>
            </ul>
          </div>
          <div class="column">
            <ul style="list-style: none; line-height: 2;">
              <li><span class="icon"><i class="fa fa-envelope"></i></span><a href="../../help/contact.html"> Contact Us</a></li>
              <li><span class="icon"><i class="fa fa-twitter"></i></span><a href="../../../twitter.com/arxiv.html"> Follow us on Twitter</a></li>
            </ul>
          </div>
        </div>
      </div>
      <!-- End Macro-Column 1 -->
      <!-- Macro-Column 2 -->
      <div class="column" style="padding: 0;">
        <div class="columns">
          <div class="column">
            <ul style="list-style: none; line-height: 2;">
              <li><a href="../../help.html">Help</a></li>
              <li><a href="../../help/policies/privacy_policy.html">Privacy Policy</a></li>
            </ul>
          </div>
          <div class="column">
            <ul style="list-style: none; line-height: 2;">
              <li><a href="https://blogs.cornell.edu/arxiv">Blog</a></li>
              <li><a href="../../help/subscribe.html"> Subscribe</a></li>
            </ul>
          </div>
        </div>
      </div>
      <!-- End Macro-Column 2 -->
    </div>

    <div class="columns" style="border-top: 1px solid #979797; margin: -0.75em;">
      <div class="column">
        <p class="help" style="margin-bottom: 0;">arXiv&#174; is a registered trademark of Cornell University.</p>
      </div>
      <div class="column">
        <p class="help" style="margin-bottom: 0;">If you have a disability and are having trouble accessing information on this website or need materials in an alternate format,
        contact <a href="mailto:web-accessibility@cornell.edu">web-accessibility@cornell.edu</a> for assistance.</p>
      </div>
    </div>
  </footer>
</body>

<!-- Mirrored from arxiv.org/list/q-bio/new by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 29 Mar 2019 22:40:35 GMT -->
</html>
